Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -0,0 +1,93 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
+import UIKit
+
+internal final class AddNewCardViewController: UIViewController {
+  @IBOutlet fileprivate weak var cardNumberLabel: UILabel!
+  @IBOutlet fileprivate weak var cardNumberTextField: UITextField!
+  @IBOutlet fileprivate weak var cardholderNameLabel: UILabel!
+  @IBOutlet fileprivate weak var cardholderNameTextField: UITextField!
+  @IBOutlet fileprivate weak var expirationLabel: UILabel!
+  @IBOutlet fileprivate weak var expirationTextField: UITextField!
+  @IBOutlet fileprivate weak var securityCodeLabel: UILabel!
+  @IBOutlet fileprivate weak var securityCodeTextField: UITextField!
+  @IBOutlet fileprivate weak var zipCodeLabel: UILabel!
+  @IBOutlet fileprivate weak var zipCodeTextField: UITextField!
+
+  private weak var saveButtonView: LoadingBarButtonItemView!
```

Comment:
Is there a reason we're holding a reference to this view?

Reply:
We need a save button on this screen and I implemented similarly to `ChangePasswordViewController` and `ChangeEmailViewController`. I can also remove this and revisit the functionality when I start on the implementation(View Model).